node/resolvers/Mutations/Users.ts

```diff
@@ -612,3 +642,45 @@ const Users = {
 }
 
 export default Users
+
+async function impersonateUser(
+  storefrontPermissionsClient: any,
+  id: string,
+  logger: any,
+  accountName: string,
+  storefrontPermissions: {
+    costcenter: { value: string }
+    userId: { value: string }
+    organization: { value: string }
+    storeUserEmail: { value: string }
+  }
+) {
+  const result = await storefrontPermissionsClient
+    .impersonateUser({ id })
```

Comment:
The name of the field that this `impersonateUser` receives is `userId`, not `id`, right? Can you fix and make sure that you test this flow after the change so we're sure it's not introducing any problems?

Reply:
Good catch!
I've changed and tested it again. I've noticed that it was working because, in the stopping impersonate flow, it sends null anyway :man_shrugging: